packages/experimental/Shimmer/src/Shimmer.tsx

```diff
@@ -36,6 +36,26 @@ export const Shimmer = compose<ShimmerType>({
     const AnimatedLinearGradient = Animated.createAnimatedComponent(LinearGradient);
     const tokens = useStyling(props).root;
 
+    const useNativeDriver: boolean = (() => {
+      // Allow the user to override the default setting if needed
+      if (props.useNativeDriver !== undefined) {
+        return props.useNativeDriver;
+      }
+      /**
+       * https://github.com/facebook/react-native/pull/29585
+       * In order for native driven animations to loop, React Native needs this fix,
+       * which is only available in React Native 0.66+, and React Native macOS 0.62+
+       */
+      const nativeVersion = Platform.constants.reactNativeVersion;
+      if (Platform.OS === 'macos' && nativeVersion.minor >= 62) {
```

Comment:
minimum version of rn-macos FURN supports is 0.63?

Reply:
True, but the check was technically ported to RN-macOS 0.62+, so I thought it was better to write it as such. 
I actually don't know if we ever hard defined minimum versions of RN FURN supports or such

Reply:
JK, we explicitly define it: https://github.com/microsoft/fluentui-react-native/blob/b8535f925f1c90edaeaee03f9a379382d786a704/packages/libraries/core/package.json#L59
I'll update this check